pkg/opentelemetry/controller.go

```diff
@@ -0,0 +1,14 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package opentelemetry
+
+import "go.opentelemetry.io/auto/pkg/probe"
+
+// OpenTelemetryController is a controller that converts probe Events to
+// OpenTelemetry spans and exports them.
+type OpenTelemetryController interface {
```

Comment:
This will stutter. i.e. `opentelemetry.OpenTelemetryController`.

```suggestion
type Controller interface {
```

Though, I'm also not sure if we even need this `opentelemtry` pacakge. For my two cents, something like `auto.Controller` seems like a better name to me.